projects/plugins/jetpack/modules/google-fonts.php

```diff
@@ -77,20 +85,22 @@ function jetpack_add_google_fonts_provider() {
 	 *
 	 * @param array $fonts_to_register Array of Google Font names to register.
 	 */
-	$fonts_to_register = apply_filters( 'jetpack_google_fonts_list', JETPACK_GOOGLE_FONTS_LIST );
+	$fonts_to_register = apply_filters( 'jetpack_google_fonts_list', $fonts );
 
 	foreach ( $fonts_to_register as $font_family ) {
+		$font_family_label = in_array( $font_family, JETPACK_GOOGLE_FONTS_DEPRECATED ) ? $font_family . ' (deprecated)' : $font_family;
+
 		wp_register_webfonts(
 			array(
 				array(
-					'font-family'  => $font_family,
+					'font-family'  => $font_family_label,
```

Comment:
`font-family` is probably needed to pass it to fonts.wp.com, and we'd need to rename the label somewhere else.

`wp_register_webfonts` probably adds them to Global styles font list, but we also need to allow font be registered so that it can be enqueued if it _actually_ is in use at the site.